xbmc/windowing/X11/GLContextEGL.cpp

```diff
@@ -541,3 +542,16 @@ void CGLContextEGL::QueryExtensions()
 
   CLog::Log(LOGDEBUG, "EGL_EXTENSIONS:{}", m_extensions);
 }
+
+int CGLContextEGL::GetBufferAge()
+{
+#ifdef EGL_BUFFER_AGE_EXT
+  EGLint age;
+  eglQuerySurface(m_eglDisplay, m_eglSurface, EGL_BUFFER_AGE_EXT, &age);
+  if (age < 1)
+    age = 2;
+  return static_cast<int>(age);
+#else
```

Comment:
Is there no runtime check necessary to check if `EGL_BUFFER_AGE_EXT` is supported (like in `CEGLContextUtils::GetBufferAge()`)?

Reply:
I guess it will result in an EGL_BAD_ATTRIBUTE error, which might leave the age undefined. I'll add a check.